src/Symfony/Component/Validator/Tests/Resources/TranslationFilesTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Resources;
+
+class TranslationFilesTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        libxml_use_internal_errors(true);
+        libxml_clear_errors();
+    }
+
+    public function testXlfFilesAreValid()
+    {
+        $translationFiles = glob(__DIR__.'/../../Resources/translations/*.xlf');
+        foreach ($translationFiles as $filePath) {
+            $xlfFile = simplexml_load_file($filePath);
+            $errors = libxml_get_errors();
+
+            if (!empty($errors)) {
+                $this->renderErrors($errors);
+            }
+
+            $this->assertEmpty($errors, sprintf('The "%s" file is not a valid XML file.', realpath($filePath)));
```

Comment:
I suggest to remove renderErrors, and replace `$this->assertEmpty(...);` by 
`$this->assertSame(array(), $errors);` so that phpunit handles the output